Tests/Configuration/appConfig.json

```
@@ -1,3 +1,5 @@
 {
-    "url": ""
+    "url": "",
+    "rnwAdminUrl":"",
+    "adminurl":""
```

Is this variable used or is it duplicate of rnwAdminUrl?
```suggestion
    "adminurl":""
```

---

there is separate feature branch , later it will merge